include/sway/config.h

```diff
@@ -110,12 +110,19 @@ struct bar_config {
 	 * Always visible in dock mode. Visible only when modifier key is held in hide mode.
 	 * Never visible in invisible mode.
 	 */
-	char *mode;
+	char *display_mode;
```

Comment:
The binding mode can probably be removed from this struct entirely. I don't think it's used anywhere.